### PR TITLE
Buffer size calculation update

### PR DESCRIPTION
**Pull request by piotrm50**
# Description of change

Buffer size is calculated by counting number of messages in the buffer instead of summing their byte sizes.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Manual tests with spammer. Unit tests.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tools/docker-network/config.docker.json [hunk 1]
```diff
@@ -54,7 +54,10 @@
     "outputPaths": [
       "stdout"
     ],
-    "disableEvents": true
+    "disableEvents": false,
+    "remotelog": {
+      "serverAddress": "192.168.0.118:5213"
+    }
```
Comment (jonastheis) [inline tools/docker-network/config.docker.json:60]: Can we put a Docker hostname here? So it would work out of the box?

File: tools/docker-network/config.docker.json [hunk 2]
```diff
@@ -70,8 +73,12 @@
     "disablePlugins": "portcheck",
     "enablePlugins": []
   },
+  "scheduler": {
+    "maxBufferSize": 150,
+    "rate": "20ms"
+  },
   "pow": {
-    "difficulty": 2,
+    "difficulty": 0,
```
Comment (jonastheis) [inline tools/docker-network/config.docker.json:81]: Why these changes and not the default values?

**Review by jonastheis — CHANGES_REQUESTED** [review]

File: tools/docker-network/config.docker.json (hunk 2 above)
Comment (piotrm50) [reply tools/docker-network/config.docker.json]: That's a badly commited file that I changed for local debugging and testing. Reverted the changes already.

**Review by jonastheis — APPROVED** [review]
